Holo-annie-jam/Screens/Menus/PauseMenuScreen.cs

```diff
@@ -11,14 +13,17 @@ class PauseMenuScreen : MenuScreen {
     /// <summary>
     /// Constructor.
     /// </summary>
-    public PauseMenuScreen()
+    public PauseMenuScreen(Action onResume)
         : base("Paused") {
         // Create our menu entries.
         MenuEntry resumeGameMenuEntry = new MenuEntry("Resume Game");
         MenuEntry quitGameMenuEntry = new MenuEntry("Quit Game");
 
         // Hook up menu event handlers.
-        resumeGameMenuEntry.Selected += OnCancel;
+        resumeGameMenuEntry.Selected += (e, args) => {
```

Comment:
instead of this, we should be overriding OnCancel to call onResume since there are other ways to exit the pause screen (including pressing ESC)